sdk/src/main/java/com/meniga/sdk/models/feed/MenigaFeed.java

```diff
@@ -73,6 +75,29 @@ public void writeToParcel(Parcel dest, int flags) {
 		dest.writeList(this);
 	}
 
+	public void moveTransactionEventsToTransaction() {
```

Comment:
If the library client does not have to perform this operation maybe we should keep it in the converter?

Reply:
True, good point